k8s_itests/deployments/paasta/fake_etc_paasta/logs.json

```
@@ -0,0 +1,10 @@
+{
+    "log_writer": {
+        "driver": "monk",
+        "options": {
+            "monk_host": "169.254.255.254",
+            "monk_port": 1473,
+            "monk_disable": true
+        }
+    }
+}
```

Maybe we might be better using the `null` log writer here

```suggestion
{
    "log_writer": {
        "driver": "null"
    }
}
```